app/models/maintenance_tasks/run.rb

```diff
@@ -34,10 +34,12 @@ class Run < ApplicationRecord
       Task.available_tasks.map(&:to_s)
     } }
     validate :csv_attachment_presence, on: :create
+    validate :verify_task_arguments_are_valid, on: :create
```

Comment:
I think I preferred `validate_task_arguments` because that was only one verb, validate vs. verify thngs are valid, which is two verbs and the noun that we care about (valid). Naming is hard 😄 fine to keep it that way anyway.